Resources/Lang/en_US.json

```diff
@@ -111,7 +111,8 @@
   "KillingMachine": "Killing Machine",
   "Escapist": "Escapist",
   "Witch": "Witch",
-  "Nemesis": "Nemesis",
+  "Mafia": "Nemesis",
```

Comment:
It would be better if the role did not have different names in the code and in the string

Reply:
Just name as Nemesis

Reply:
> It would be better if the role did not have different names in the code and in the string

mk, there was just some strings in chatmanager that I was unsure were it pulled that from. But I can try to change it if it's very important. I'm refering to a "after", "Before" thing.

Reply:
okay I found it, will change now.

Reply:
done